java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/analyser/deploy/DeployCommand.java

```diff
@@ -103,154 +126,145 @@ public DeployCommand(BeanManager beanManager,
 
     public void run() {
         try {
-            java.nio.file.Path modified = Files.createTempFile("deployment", ".tar.gz");
-            try {
```

Comment:
I think this was left over from https://github.com/redhat-appstudio/jvm-build-service/pull/477